ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -209,6 +210,24 @@
 /* End PBXResourcesBuildPhase section */
 
 /* Begin PBXShellScriptBuildPhase section */
+		30531896244020B4005AE004 /* ShellScript */ = {
```

Comment:
Is the script for Xcode 10? After 30th April, apps that submitted to App Store must use iOS 13 SDK or later which is Xcode 11+.

Reply:
That was auto generated, so no idea.